apps/server/src/modules/files-storage/uc/file-record.uc.ts

```diff
@@ -18,31 +18,50 @@ export class FileRecordUC {
 		private readonly authorizationService: AuthorizationService
 	) {}
 
-	async patchFilename(userId: EntityId, params: SingleFileParams, data: RenameFileParams) {
+	private checkDouplicatedNames(fileRecords: FileRecord[], data: RenameFileParams): void | ConflictException {
+		if (fileRecords.find((item) => item.name === data.fileName)) {
+			throw new ConflictException('FILE_NAME_EXISTS');
+		}
+	}
+
+	private modifiedFileName(
```

Comment:
"modifiedFileName" doesn't sound like a method name. Wouldn't "setEntityName" be correct?

Reply:
I use set first, but we put the entity to the method and modified one of the values for it. For this i want to find a name set it give the developer the hind that the value is not returned, the entity is not returned but something is different after execution.

Reply:
fixed